src/nal/sps.rs

```diff
@@ -1414,6 +1433,20 @@ mod test {
             seq_parameter_set_id: SeqParamSetId::from_u32(0).unwrap(),
             chroma_info: ChromaInfo{
                 chroma_format: ChromaFormat::YUV420,
+                scaling_matrix: Some(SeqScalingMatrix {
+                    scaling_list4x4: vec![
+                        ScalingList::List(Box::new([16; 16])),
+                        ScalingList::List(Box::new([16; 16])),
+                        ScalingList::List(Box::new([16; 16])),
+                        ScalingList::List(Box::new([16; 16])),
+                        ScalingList::List(Box::new([16; 16])),
+                        ScalingList::List(Box::new([16; 16])),
+                    ],
+                    scaling_list8x8: vec![
+                        ScalingList::NotPresent,
+                        ScalingList::NotPresent,
+                    ]
+                }),
                 ..ChromaInfo::default()
```

Comment:
It'd be nice to have a test case that exercises the new code paths more fully.

Reply:
I modified the test case a bit, are there things I forgot to include there that you wish were covered?